src/Sarif.FunctionalTests/SarifConverterTestData/AndroidStudio/AndroidStudio_issueLog3_raw.xml.sarif

```diff
@@ -5,21 +5,28 @@
       "toolInfo": {
         "name": "AndroidStudio"
       },
+      "runInfo": {
+        "fileInfo": {
+          "app/src/main/java/microsoft/androidstudioformatexample/MainActivity.java": [
+            {
+              "uri": "app/src/main/java/microsoft/androidstudioformatexample/MainActivity.java",
+              "mimeType": "text/x-java-source"
+            }
+          ]
+        }
+      },
       "results": [
         {
           "ruleId": "'for' loop replaceable with 'foreach'",
           "fullMessage": "<code>for</code> loop replaceable with 'foreach' #loc",
```

Comment:
#1 No, it's not ok. We shouldn't assume the client can parse HTML. In fact the client _shouldn't_ parse HTML, it's just asking for trouble.

I opened sarif-standard/sarif-spec#136, "Add guidance: No formatting information in message and description properties".